proto/viam/app/data/v1/data.proto

```diff
@@ -53,19 +53,22 @@ service DataService {
   rpc BoundingBoxLabelsByFilter(BoundingBoxLabelsByFilterRequest) returns (BoundingBoxLabelsByFilterResponse);
 }
 
+// Order specifies the order in which data is returns.
 enum Order {
   ORDER_UNSPECIFIED = 0;
   ORDER_DESCENDING = 1;
   ORDER_ASCENDING = 2;
 }
 
+// DataRequest encapsulates the filter for the data, a limit on the maximum results returned, a last string associated with the last returned document, and the sorting order by time.
```

Comment:
Might be worth explaining `last` a bit more since have had to explain that to folks a few times. This could include an example of where this is returned in the API and how it's passed back into here in order to retrieve data coming after `last`

Reply:
Thanks for adding! [nit] would make the line widths on this documentation shorter